app/models/fax_recipient.rb

```diff
@@ -12,12 +12,23 @@ def number
   end
 
   def name
-    ENV.fetch("FAX_RECIPIENT", "Michigan Bridges")
+    office["name"].to_s.titleize
+  end
+
+  def office
+    if office_location.present?
+      office = self.class.offices[office_location]
+      office["name"] = office_location
+      office
+    else
+      self.class.find_by(zip: residential_address.zip)
+    end
   end
 
   def self.find_by(zip:)
     name = covered_zip_codes[zip] || configuration["fallback_office"]
     Rails.logger.debug("Mapped zip #{zip} to office #{name}")
+    offices[name]["name"] = name
```

Comment:
this feels a bit confusing -- why this change?

Reply:
This was hella annoying. It's because our yml file doesn't include the name of the office in the data, just as the key. It would be better to do as part of processing the YML file initially, not this far downstream.

Reply:
interesting -- so is there a test we can write that confirms that change does what we want?